helm/azure-resourcemanager-exporter/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
+{{- if semverCompare ">=1.14.0-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Can you do the same that done for `Ingress` version in `templates/_helpers.tpl`?

Reply:
Ingress object is not like RBAC they moved it from extensions/v1beta1 to  networking.k8s.io/v1beta1 and clusters can have networking.k8s.io/v1beta1 api-version without ingress inside it